api/nakadi-event-bus-api.yaml

```diff
@@ -479,7 +475,40 @@ paths:
                 Number of seconds the client ought to wait before making a follow-up request.
               type: number
 
-  /event-types/{name}/partitions:
+  '/event-types/{name}/schemas':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        List of schemas ordered from most recent to oldest.
+    responses:
+        '200':
+          schema:
+            type: object
+            properties:
+              items:
+                description: list of schemas.
+                type: array
+                items:
+                  $ref: '#/definitions/EventTypeSchema'
+            required:
+              - items
+
+  '/event-types/{name}/schemas/{version}':
```

Comment:
@dehora every event will be enriched with the version string. For example "1.0.0".